src/utils/fake-backend.js

```diff
@@ -31,6 +31,7 @@ export function configureFakeBackend() {
 
       function addProduct() {
         const product = body;
```

Comment:
i would remove this assignment and just use body in the next line to pull out the id